mod/display.php

```diff
@@ -257,28 +256,29 @@ function display_content(App $a, $update = false, $update_uid = 0)
 
 	$is_remote_contact = false;
 	$item_uid = local_user();
+	$page_uid = 0;
 
 	$parent = null;
 	if (!empty($parent_uri_id)) {
 		$parent = Post::selectFirst(['uid'], ['uri-id' => $parent_uri_id, 'wall' => true]);
 	}
 
 	if (DBA::isResult($parent)) {
-		$a->profile['uid'] = ($a->profile['uid'] ?? 0) ?: $parent['uid'];
-		$is_remote_contact = Session::getRemoteContactID($a->profile['uid']);
+		$page_uid = ($page_uid ?? 0) ?: $parent['uid'];
```

Comment:
Parentheses are superfluous with binary operators like `??` and `?:` since they don't have the ambiguity of the ternary operator.
```suggestion
		$page_uid = $page_uid ?? 0 ?: $parent['uid'];
```